pkgs/development/interpreters/php/default.nix

```diff
@@ -19,7 +19,8 @@ let
 
       enableParallelBuilding = true;
 
-      buildInputs = [ flex bison pkgconfig systemd ];
+      buildInputs = [ flex bison pkgconfig ]
+      ++ lib.optional stdenv.isLinux [systemd ];
 
```

Comment:
@peterhoeg no problem.
@vbgl thx for the lib.optional tip, i'll amend my commit in short